src/dash-playlist-loader.js

```diff
@@ -791,9 +794,10 @@ export default class DashPlaylistLoader extends EventTarget {
       this.trigger('playlistunchanged');
     }
 
-    if (!this.media().endList) {
+    if (!this.media().endList && !this.mediaUpdateTimeout) {
       this.mediaUpdateTimeout = window.setTimeout(() => {
         this.trigger('mediaupdatetimeout');
+        this.mediaUpdateTimeout = null;
```

Comment:
Will this clear out `this.mediaUpdateTimeout` only after the call to `refreshMedia_`? Would the check in the conditional above then block the new timeout?

Reply:
You are right. I just realized this with some testing. Using the (netlify deploy to test some drm that requires https with these changes)